modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf

```diff
@@ -1,9 +1,10 @@
+
 process GATK4_DETERMINEGERMLINECONTIGPLOIDY {
     tag "$meta.id"
     label 'process_single'
 
     //Conda is not supported at the moment: https://github.com/broadinstitute/gatk/issues/7811
-    container "nf-core/gatk:4.4.0.0" //Biocontainers is missing a package
+    container "quay.io/nf-core/gatk:4.4.0.0" //Biocontainers is missing a package
```

Comment:
```suggestion
    container "nf-core/gatk:4.4.0.0" //Biocontainers is missing a package
```
Quay.io is the default registry in all nf-core pipelines so we leave this out for more flexibility

Reply:
When I tried without it, singularity fails to pull the image. https://github.com/nf-core/modules/actions/runs/5394391319/jobs/9795497440

Reply:
@adamrtalbot can you help with this? :)

Reply:
Awesome thank you!

Reply:
Cool! I will let `quay.io` be a part of the uri for now, but I will remove it after the next tools release 😄